src/helpers.ts

```diff
@@ -1,8 +1,39 @@
+const fontWeightKeywords = require('css-font-weight-keywords');
 const cssFontSizeKeywords = require('css-font-size-keywords');
+const fontStretchKeywords = require('css-font-stretch-keywords');
 
 export function isSize(value: string) {
-	return /^[\d\.]/.test(value)
+	return /^\+?[\d\.]/.test(value)
 		|| value.indexOf('/') !== -1
 		|| cssFontSizeKeywords.indexOf(value) !== -1
 	;
 }
+
+export function isWeight(value: string) {
+	return fontWeightKeywords.indexOf(value) !== -1
+		|| isOnlyNumber(value, false);
+}
+
+export function isStretch(value: string) {
+	return fontStretchKeywords.indexOf(value) !== -1
+		|| isOnlyNumber(value, true);
+}
+
+function isOnlyNumber(value: string, isPercent: boolean): boolean {
+	const match = /^(\+|-)?(\d|\.)/.exec(value);
```

Comment:
I am not sure about this (changing the regular expression):

 * as to the first question, this regular expression is only a pre-test: any non-number value would get rejected at `parseFloat(val).toString() === val`

 * this regular expression is also used for determining if there is leading zero missing for fraction-only values (e.g. `.24`): in this case a zero has to be prepended, so that the test `parseFloat(val).toString() === val` works

-> so changing the regular expression would also require to re-implement the test (for checking if the string really only contains a number, or something more, or something else entirely)

Reply:
Either way, for readability, changing the variable names to `sign`, `value` and `percent` would be much more clear.